src/transforms/select.jl

```diff
@@ -42,6 +82,8 @@ function apply(transform::Select, table)
   select  = _select(transform.cols, allcols)
   reject  = setdiff(allcols, select)
 
+  @assert select ⊆ Tables.columnnames(table)
```

Comment:
Maybe we should also assert that the select is not empty? I think this is still possible right?

Reply:
Done!
Resolved in 6199b20.